CHANGES/9136.feature

```diff
@@ -0,0 +1,2 @@
+Updated minimum supported version of Debian to 11 (Bullseye). 
```

Comment:
Perhaps we need a "Removal" as well?

I mean, you could do that in a separate commit where you clean up the debian 10 workarounds (referenced in another comment) in ./pulp_devel/tasks/install_podman.yml .